console/src/app/pages/users/user-list/user-table/user-table.component.ts

```diff
@@ -194,6 +194,9 @@ export class UserTableComponent implements OnInit {
         case 'username':
           sortingField = UserFieldName.USER_FIELD_NAME_USER_NAME;
           break;
+        case 'preferredLoginName':
+          sortingField = UserFieldName.USER_FIELD_NAME_USER_NAME;
```

Comment:
```suggestion
          // TODO: replace with preferred username sorting once implemented
          sortingField = UserFieldName.USER_FIELD_NAME_USER_NAME;
```

Reply:
Hope I can fix this TODO in the future. Thanks @peintnermax